build.cake

```diff
@@ -0,0 +1,50 @@
+var target = Argument("target", "Default");
+var configuration = Argument("configuration", "Release");
+
+var artifactsDir = "./artifacts/";
+var projectFile = "./src/Grok.Net/Grok.Net.csproj";
+var binDir = "./src/Grok.Net/bin/";
+var objDir = "./src/Grok.Net/obj/";
+
+Task("Clean")
+    .Does(() =>
+{
+    CleanDirectories(new[]{ artifactsDir, binDir, objDir });
+});
+
+Task("Restore-NuGet-Packages")
+    .IsDependentOn("Clean")
+    .Does(() =>
+{
+    DotNetCoreRestore(projectFile);
+});
+
+Task("Build")
+    .IsDependentOn("Restore-NuGet-Packages")
+    .Does(() =>
+{
+    DotNetCoreBuild(projectFile, new DotNetCoreBuildSettings(){ Configuration = configuration });
+});
+
+Task("Run-Unit-Tests")
+    .IsDependentOn("Build")
+    .Does(() =>
+{
+    // No unit tests yet!
+});
+
+Task("NuGet-Pack")
+    .IsDependentOn("Run-Unit-Tests")
+    .Does(() =>
+{
+    DotNetCorePack(projectFile, new DotNetCorePackSettings()
```

Comment:
The same here. `DotNetCorePack` runs nuget restore command and build command. In this task, we have already restored and built a project. Try this one:
```csharp
var settings = new DotNetCorePackSettings
{
	Configuration = configuration,
	NoRestore = false,
	NoBuild = true,
	OutputDirectory = artifactsDir
};

DotNetCorePack(projectFile, settings);
```

Reply:
Done 👍